lib/sycamore/sycamore/docset.py

```diff
@@ -835,7 +835,7 @@ def llm_filter(
         llm: LLM,
         new_field: str,
         prompt: Union[list[dict], str],
-        field: Optional[str] = "text_representation",
```

Comment:
This feels like an unrelated change. It's generally best to have those in separate PRs, but if necessary make sure it's included in the description.

Reply:
Moved it to a separate PR: https://github.com/aryn-ai/sycamore/pull/720

Reply:
It's still showing up in my diff, but maybe it's github diff weirdness.

Reply:
Yeah that must be it, the PR above is merged.